app/src/main/java/com/google/android/material/appbar/FlingBehavior.java

```diff
@@ -68,6 +84,26 @@ public boolean onInterceptTouchEvent(final CoordinatorLayout parent, final AppBa
         return super.onInterceptTouchEvent(parent, child, ev);
     }
 
+    @Override
+    public boolean onStartNestedScroll(@NotNull final CoordinatorLayout parent,
```

Comment:
Please use AndroidX's `NonNull` annotations instead of JetBrains' `NotNull` annotations